Engines/Wine/Verbs/FAudio/script.js

```diff
@@ -10,7 +10,7 @@ include("utils.functions.filesystem.files");
 * @returns {Wine} Wine object
 */
 Wine.prototype.faudio = function (faudioVersion) {
-    if(typeof faudioVersion !== 'string') 
+    if (typeof faudioVersion !== 'string')
```

Comment:
Blame @ImperatorS79 !

Reply:
No reason to blame people who try to help you!

Reply:
@plata You blame ppl in linux if they do something wrong so that they know about it.. it's the way we QA each other and i encourage you to do the same.

Reply:
@Kreyren There is a big difference between destructive / personal attack and code review / constructive remarks and quality assurance. Please beware of it.